docs/src/main/sphinx/connector/mysql.rst

```diff
@@ -80,6 +80,30 @@ property, which can be set to ``UNNECESSARY`` (the default),
 
 .. include:: jdbc-type-mapping.fragment
 
+.. _mysql-sql-support:
+
+SQL support
+-----------
+
+The connector provides read access and write access to data and metadata in the
+MySQL database. In addition to the :ref:`globally available <sql-globally-available>` and
+:ref:`read operation <sql-read-operations>` statements, the connector supports
+the following features:
+
+* :doc:`/sql/insert`
+* :doc:`/sql/delete`
+* :ref:`sql-schema-table-management`
+
+.. include:: sql-delete-limitation.fragment
+
+The following SQL statements are not yet supported:
+
+* :doc:`/sql/grant`
+* :doc:`/sql/revoke`
+* :doc:`/sql/show-grants`
+* :doc:`/sql/show-roles`
+* :doc:`/sql/show-role-grants`
+
```

Comment:
Remove. The SQL support template you're adding is intended to replace this content.